exp-player/addon/components/exp-rating-form/component.js

```diff
@@ -4,569 +4,580 @@ import layout from './template';
+                type: 'input',
+                description: 'measures.questions.12.label',
+                value: null,
+                optional: true
+            }]
+    },
+    generateSchema({
+        question: 'measures.questions.13.label',
+        type: 'radio',
+        page: 6,
+        items: items['13'],
+        scale: [
+            'measures.questions.13.options.disagreeStrongly',
+            'measures.questions.13.options.disagree',
+            'measures.questions.13.options.neutral',
+            'measures.questions.13.options.agree',
+            'measures.questions.13.options.agreeStrongly'
+        ],
+        options: {labelTop: true}
+    }),
+    generateSchema({
+        question: 'measures.questions.14.label',
+        type: 'radio',
+        page: 6,
+        items: items['14'],
+        scale: [
+            'measures.questions.14.options.disagreeStrongly',
+            'measures.questions.14.options.disagree',
+            'measures.questions.14.options.neutral',
+            'measures.questions.14.options.agree',
+            'measures.questions.14.options.agreeStrongly'
+        ],
+        options: {labelTop: true}
+    }),
+    generateSchema({
+        question: 'measures.questions.15.label',
+        type: 'radio',
+        page: 6,
+        items: items['15'],
+        scale: [
+            'measures.questions.15.options.disagreeStrongly',
+            'measures.questions.15.options.disagree',
+            'measures.questions.15.options.neutral',
+            'measures.questions.15.options.agree',
+            'measures.questions.15.options.agreeStrongly'
+        ],
+        options: {labelTop: true}
+    }),
+    generateSchema({
+        question: 'measures.questions.16.label',
+        type: 'radio',
+        page: 7,
+        items: items['16'],
+        scale: [
+            'measures.questions.16.options.notAtAll',
+            'measures.questions.16.options.aLittle',
+            'measures.questions.16.options.quiteaBit',
+            'measures.questions.16.options.completely'
+        ],
+        options: {labelTop: true}
+    }),
+    generateSchema({
+        question: 'measures.questions.17.label',
+        type: 'radio',
+        page: 7,
+        items: items['17'],
+        scale: [
+            'measures.questions.17.options.disagreeStrongly',
+            'measures.questions.17.options.disagree',
+            'measures.questions.17.options.neutral',
+            'measures.questions.17.options.agree',
+            'measures.questions.17.options.agreeStrongly'
+        ],
+        options: {labelTop: true}
+    })
 ];
 
 const Validations = buildValidations(generateValidators(questions));
 
 export default ExpFrameBaseComponent.extend(Validations, {
     type: 'exp-rating-form',
     layout: layout,
-    questions: Ember.computed(function() {
+    currentPage: 1,
+    totalPages: 7,
+    progressBarPage: Ember.computed('currentPage', function () {
+        return this.currentPage + 4;
```

Comment:
There are four pages before the rating form, and the progress bar indicates how far you are in taking the entire 11-page survey, so I needed two page counts. One for the rating form pagination and one for the page number with respect to the whole survey.